stages/org.osbuild.pki.update-ca-trust

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+import os
+import sys
+import uuid
+
+import osbuild.api
+
+
+def main(tree, options):
+    for cert in options["certs"]:
+        rnd = str(uuid.uuid4())
```

Comment:
I'm curious why we use a uuid4 here, maybe a small comment? I also wonder if this really should be random or if we could use e.g. the hash of the input cert (which is IIRC what most certs from ca-certificates do) so that we have predictable filenames for the certs.

Reply:
Sure, let’s use SHA256 sum.